.github/workflows/individual-image-scanner-quay.yaml

```diff
@@ -0,0 +1,159 @@
+---
+name: Fetch security scan results from quay
+on:
+  push:
+    branches:
+      - clair
+  workflow_dispatch:
```

Comment:
There must be a `pull_request` trigger here according to [this comment](https://github.com/openshift-pipelines/pipeline-service/pull/204#r954583381). We will be changing the branches to `main` right?

Reply:
ah, clair was a typo. Was testing on that and forgot to update to main. Done.
The link is not redirecting to any comment. But, I think we want to run the vulnerability check once the PR is merged. Not for a Pull Request. Happy to hear other's thoughts on this as well.

Reply:
I think that too, once PR is merged then only check should start.

Reply:
Images are built on push not on PR. I am open to have this behavior changed but that would be a prerequisite. Bharath, I haven't looked at the details but how do you ensure that you are retrieving the scan result of the image that has been built after merge of the PR and not the previous one, i.e that the actions are sequential or that you wait till the new result is available?

Reply:
I think adding `concurrency` to the GitHub Actions might help. See https://docs.github.com/en/actions/using-jobs/using-concurrency

Reply:
Concurrency is one thing but I am not too worried with having multiple PRs getting merged at the same time. We don't have that many volumes. The thing is that jjobs need to happen sequentially. [workflow_run](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run) may help but may still not be enough. IIRC clair scanning is done async.

Reply:
For now, we don't have that level of granular control. We could use workflow_run or workflow_dispatch but I would prefer to do this once we unify 6 different GitHub actions we currently have for building images into one using paths_filter that I have used here. 
That way, we could trigger scan action once build action is completed. WDYT?

p.s.: I think we might be able to control this better when we have PipelineRuns instead of actions.

Reply:
Yes I am fine either ways. Until we move to PaC what is important to me is that we lower the noise ratio:
- when a PR is created we get the CVEs for +/- the changes that have been introduced by it
- we don't need a subsequent PR to figure out CVEs introduced by a previous PR

Reply:
Ack. If there are no other changes, then how about merging this PR?
Next actions:
1. Merge individual actions into one single action for building and uploading images.
2. Introduce workflow_run/workflow_dispatch to ensure sync between image uploads and scan results.